<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar</class>
 <widget class="QWidget" name="toolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QPushButton" name="open_file">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X Axis Parameter</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y Axis Parameter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeView" name="file_display"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>File Browser</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="add_Y">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Y Parameter</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="qspinparam" name="x_select"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QScrollArea" name="multiY">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>68</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Additional Y Parameters</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="qspinparam" name="y_select"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qspinparam</class>
   <extends>QSpinBox</extends>
   <header>qspinparam.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
